### DISCUSSION TITLE
✨ [Feature]: 完善InputNumber组件功能

### DISCUSSION THREAD

**Discussion by kagol** [DevCloudFE/vue-devui#1386] [General]
### What problem does this feature solve

InputNumber组件目前还有以下功能未完成：
- [x] 基本功能
- [x] 禁用状态
- [x] 允许空值
- [x] 设置最大长度
- [x] 正则限制 Finished by @vaebe on #1207 
- [ ] 限制小数

参考：
https://devui.design/components/zh-cn/input-number/demo

### What does the proposed API look like

参考：
https://devui.design/components/zh-cn/input-number/demo

**Comment by vaebe** [comment]
正则限制、限制小数 我觉得我可以搞一搞

**Comment by fu050409** [comment]
限制小数已经在 #1865 提交pr
